### ISSUE TITLE
Link stopped working

### ISSUE THREAD

**Issue by smileart** [rgrove/rawgit#83]
Some time ago I created [this link](https://rawgit.com/smileart/6cf5de61d5f393d26fa0/raw/) on `rawgit.com` which is supposed to link [this gist](https://gist.github.com/smileart/6cf5de61d5f393d26fa0) but somehow it stopped working and started to show 404 error. When I try to create a new one, the field on the site become red. I don't want to lose this link cause it was shortened on the `bit.ly` as [http://bit.ly/vim-min](http://bit.ly/vim-min). Wuld be great to understand what's wrong or what I'm doing wrong.

**Comment by rgrove** [comment]
Ugh, sorry about that!

That link actually should never have worked, because RawGit always needs a filename with extension in order to figure out the correct `Content-Type`, but it looks like there was a time when we were unintentionally redirecting gist URLs without filenames directly to `gist.githubusercontent.com`, which would in turn redirect to the appropriate URL if there was only a single file in the gist.

When we added support for defaulting to `index.html` on URLs without filenames in #73, this accidental redirect behavior stopped working and instead we began serving 404s for gists without an `index.html` file that are requested without a filename in the URL.

I've added a hard-coded redirect to make your old bit.ly link work again.
